### PR TITLE
Init 2.414.1

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

**Review by jglick — APPROVED** [review]

**Comment by jglick** [comment]
@MarkEWaite please just double-check that the `full-test` label, applied after filing the PR, is honored. I think it should be because https://github.com/jenkinsci/bom/blob/9f3c89247e2f6c141666413dae7883f6691b32e2/Jenkinsfile#L84 is checked after the `prep` phase and I believe the label lookup uses the current state of the PR rather than the state at the time the build started.

**Comment by MarkEWaite** [comment]
> @MarkEWaite please just double-check that the `full-test` label, applied after filing the PR, is honored.

Will do.  I won't merge this until we have a confirmed pass of `full-test` with actual execution of the tests.

**Comment by jglick** [comment]
I just checked and it is running PCT, so automerge should be fine.

**Comment by jglick** [comment]
Do you plan to release this? https://github.com/jenkinsci/ec2-plugin/pull/875 would apparently use it (@car-roll).

**Comment by MarkEWaite** [comment]
Happy to release it today or tomorrow.

File: sample-plugin/pom.xml [hunk 1]
```diff
@@ -761,11 +761,18 @@
...
+        <bom>2.414.x</bom>
+        <jenkins.version>2.414</jenkins.version>
+      </properties>
+    </profile>
     <profile>
       <id>2.401.x</id>
       <properties>
         <bom>2.401.x</bom>
-        <jenkins.version>2.401.1</jenkins.version>
+        <jenkins.version>2.401.3</jenkins.version>
```
Comment (jglick) [inline sample-plugin/pom.xml:775]: I think according to the above logic this should remain 2.401.1 until 2.414.1 is released. At this point it does not much matter.

**Pull request merged**